src/components/CompareResults/ResultsView.tsx

```diff
@@ -1,7 +1,15 @@
 import { useEffect } from 'react';
+import React from 'react';
```

Comment:
nit: import `Suspense` and `useState` directly for a better tree shaking operation

Reply:
Yeah that was cleanup I forgot to do.